src/renderer/views/Playlist/Playlist.js

```diff
@@ -138,8 +219,6 @@ export default defineComponent({
     },
 
     getPlaylistInvidious: function () {
-      this.isLoading = true
-
       invidiousGetPlaylistInfo(this.playlistId).then((result) => {
         this.infoData = {
```

Comment:
`infoData` doesn't seem to be used anymore, so any code relating to it should probably be removed.